$MirOS: ports/security/gpgme/patches/patch-tests_gpg_Makefile_am,v 1.1 2006/09/17 20:35:08 tg Exp $
--- tests/gpg/Makefile.am.orig	Fri Oct  7 13:16:12 2005
+++ tests/gpg/Makefile.am	Tue Mar 20 17:44:46 2007
@@ -25,11 +25,19 @@ TESTS_ENVIRONMENT = GNUPGHOME=. GPG_AGEN
 
 # The keylist tests must come after the import and the edit test.
 noinst_HEADERS = t-support.h
+if HAVE_PTHREAD
 TESTS = t-encrypt t-encrypt-sym t-encrypt-sign t-sign t-signers	\
 	t-decrypt t-verify t-decrypt-verify t-sig-notation \
 	t-export t-import t-trustlist t-eventloop t-edit \
 	t-keylist t-keylist-sig t-thread1 t-wait t-encrypt-large \
 	t-file-name
+else
+TESTS = t-encrypt t-encrypt-sym t-encrypt-sign t-sign t-signers	\
+	t-decrypt t-verify t-decrypt-verify t-sig-notation \
+	t-export t-import t-trustlist t-eventloop t-edit \
+	t-keylist t-keylist-sig t-wait t-encrypt-large \
+	t-file-name
+endif
 
 CLEANFILES = secring.gpg pubring.gpg trustdb.gpg
 DISTCLEANFILES = pubring.gpg~ random_seed gpg.conf
@@ -41,7 +49,9 @@ INCLUDES = -I$(top_srcdir)/gpgme
 
 AM_CPPFLAGS = @GPG_ERROR_CFLAGS@
 LDADD = ../../gpgme/libgpgme.la
+if HAVE_PTHREAD
 t_thread1_LDADD = ../../gpgme/libgpgme-pthread.la
+endif
 
 # We don't run t-genkey in the test suite, because it takes too long
 noinst_PROGRAMS = $(TESTS) t-genkey
